requirements/ci.in

```diff
@@ -1,6 +1,6 @@
 # Requirements for running tests in Travis
 -c constraints.txt
 
-codecov                   # Code coverage reporting
```

Comment:
I think we don't need `coverage` here, since tox will manage its own virtualenvs.

Reply:
I don't understand, don't we need coverage so that reports can be generated as a part of testing?  My understanding was that we still need to generate coverage reports and then the github action will upload them.

Reply:
Yep! But it's in test.txt, and tox installs text.txt into its own virtualenv before running pytest. The ci.txt just installs tox itself at this point.

Reply:
Gotcha, pytest-cov is in `test.in` and so that will pull in codcov, we don't need it in `ci.in`